controller.py

```diff
@@ -565,9 +555,16 @@ class URNLookupController(CoreURNLookupController):
     log = logging.getLogger("URN lookup controller")
 
     def __init__(self, _db):
-        self.registrar = IdentifierResolutionRegistrar(_db)
+        self.default_collection_id = None
         super(URNLookupController, self).__init__(_db)
 
+    @property
+    def default_collection(self):
```

Comment:
This (and the TODO comment below) are prep work for when unaffiliated identifiers are welcome again.